init.sh

```diff
@@ -87,28 +116,34 @@ cd "$DIR_LIB_GEN" && [ ! -f "$REPLACECALL_JAR" ] \
 #
 echo
 echo "Setting up Gradle dependencies ... "
-GRADLE_ZIP=defects4j-gradle.zip
-# The BSD version of stat does not support --version or -c
-if stat --version &> /dev/null; then
-    # GNU version
-    cmd="stat -c %Y $GRADLE_ZIP"
-else
-    # BSD version
-    cmd="stat -f %m $GRADLE_ZIP"
-fi
+
+GRADLE_DISTS_ZIP=defects4j-gradle-dists.zip
+GRADLE_DEPS_ZIP=defects4j-gradle-deps.zip
+
+dists_ts=$(get_modification_timestamp $GRADLE_DISTS_ZIP)
+deps_ts=$(get_modification_timestamp $GRADLE_DEPS_ZIP)
 
 cd "$DIR_LIB_GRADLE"
-if [ -e $GRADLE_ZIP ]; then
-    old_ts=$($cmd)
-else
-    old_ts=0
+
+old_dists_ts=0
+old_deps_ts=0
+
+if [ -e $GRADLE_DISTS_ZIP ]; then
+    old_dists_ts=$($dists_ts)
 fi
+if [ -e $GRADLE_DEPS_ZIP ]; then
+    old_deps_ts=$($deps_ts)
+fi
+
 # Only download archive if the server has a newer file
-wget -N http://people.cs.umass.edu/~rjust/defects4j/download/$GRADLE_ZIP
-new=$($cmd)
+wget -N $HOST_URL/$GRADLE_DISTS_ZIP
+wget -N $HOST_URL/$GRADLE_DEPS_ZIP
+new_dists_ts=$($dists_ts)
```

Comment:
See comment above; I think you need to call the get_modification_timestamp function.